models/views/scratch/snowplow_unified_screen_summary_metrics.sql

```diff
@@ -0,0 +1,61 @@
+{#
+Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
```

Comment:
Maybe not important but isn't this technically 2024-present?

Reply:
Good point, have changed to 2024!